packages/splits-sdk/src/client/liquidSplit.ts

```diff
@@ -123,17 +120,19 @@ class LiquidSplitTransactions extends BaseTransactions {
   }: DistributeLiquidSplitTokenConfig): Promise<TransactionFormat> {
     validateAddress(liquidSplitAddress)
     validateAddress(token)
-    if (this._shouldRequreWalletClient) this._requireWalletClient()
+    if (this._shouldRequireWalletClient) this._requireWalletClient()
 
     const distributorPayoutAddress = distributorAddress
       ? distributorAddress
       : this._walletClient?.account
       ? this._walletClient.account.address
-      : ADDRESS_ZERO
+      : zeroAddress
     validateAddress(distributorPayoutAddress)
 
-    // TO DO: handle bad split id/no metadata found
-    const { holders } = await this.getLiquidSplitMetadata({
+    this._requireDataClient()
```

Comment:
ideally we have a provider only version of all the get metadata functions. so if there's no api key we fall back to it (and maybe even if our server returns a 5xx error we fallback to it?).

if that's not ready yet all good, think we can launch without it and then add afterwards.

Reply:
yeah, I wanted to descope a few things, this being one of them in this PR